ui/plugin-system/src/utils/variables.ts

```diff
@@ -37,12 +44,14 @@ export function replaceTemplateVariable(text: string, varName: string, templateV
     replaceString = templateVariableValue;
   }
 
-  return text.replaceAll(variableTemplate, replaceString);
+  text = text.replaceAll(variableTemplate, replaceString);
+  return text.replaceAll(bracketsVariableTemplate, replaceString);
 }
 
 // This regular expression is designed to identify variable references in a template string.
```

Comment:
comment to update mentioning the 3 possibilities, or maybe just amend the second one to say that the second part `:value` is optional maybe.

Reply:
Updated, wdyt?